packages/ckeditor5-dev-changelog/src/utils/removescope.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * @license Copyright (c) 2003-2025, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import type { ParsedFile } from '../types';
+
+export function removeScope( parsedChangesetFiles: Array<ParsedFile> ): Array<ParsedFile> {
+	return parsedChangesetFiles.map( changeset => {
+		delete changeset.data.scope;
```

Comment:
Delete still modifies the original object. 
You would have to either copy the whole array, or construct a new object without the unwanted prop:
```
const removeProperty = (prop) => (obj) => {
  const { [prop]: propToRemove, ...rest } = obj;
  return rest;
};
```